.travis.yml

```diff
@@ -17,62 +17,21 @@ jobs:
         - DESC="acceptance tests"
         - CMD="./mvnw install --projects acceptance-tests --also-make --activate-profiles all --no-transfer-progress"
 
-    - jdk: oraclejdk8
-      env:
-        - DESC="findbugs"
-        - CMD="./mvnw install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: oraclejdk8
-      env:
-        - DESC="checkstyle"
-        - CMD="./mvnw install checkstyle:check --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: oraclejdk8
-      env:
-        - DESC="unit tests"
-        - CMD="./mvnw install"
-
     - jdk: oraclejdk8
       env:
         - DESC="compile performance-tests"
         - CMD="./mvnw install --projects performance-tests --also-make --activate-profiles all -DskipTests=true --no-transfer-progress"
 
-    - jdk: oraclejdk11
-      env:
-        - DESC="compile javadoc"
-        - CMD="./mvnw install javadoc:jar --projects 'eclipse-collections-code-generator,eclipse-collections-code-generator-maven-plugin,eclipse-collections-api,eclipse-collections,eclipse-collections-testutils,eclipse-collections-forkjoin' --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: openjdk11
-      env:
-        - DESC="unit tests openjdk11"
-        - CMD="./mvnw install --no-transfer-progress"
-
     - jdk: oraclejdk11
       env:
         - DESC="unit tests oraclejdk11"
         - CMD="./mvnw install --no-transfer-progress"
 
-    - jdk: openjdk14
-      env:
-        - DESC="unit tests openjdk14"
-        - CMD="./mvnw install --no-transfer-progress"
-
     - jdk: oraclejdk14
       env:
         - DESC="unit tests oraclejdk14"
         - CMD="./mvnw install --no-transfer-progress"
 
-    - jdk: openjdk-ea
-      env:
-        - DESC="unit tests openjdk-ea"
-        - CMD="./mvnw install --no-transfer-progress"
-
-    - jdk: oraclejdk8
```

Comment:
Please leave the unit tests for Java15-EA for now.

Reply:
This is the same as below :)

Reply:
Oh, I see. Thanks for clarifying.

Reply:
I mean we have the same test running in the `allow_failures` section.

Reply:
That’s now how Travis works. Travis needs the actual build and also a section to allow failures. The allow failures has to be a copy paste for Travis to “allow failure”. Please revert this

Reply:
Oops is it? That looks strange.  I didn't know that. Okay will revert that for now.

Reply:
Yeah took me about 20 iterations to figure out what I was missing when I set them up initially!